java/src/jmri/jmrit/dispatcher/ActivateTrainFrame.java

```diff
@@ -1152,7 +1152,7 @@ private void trainInfoToDialog(TrainInfo info) {
     private TrainInfo dialogToTrainInfo() {
         TrainInfo info = new TrainInfo();
         info.setTransitName((String) transitSelectBox.getSelectedItem());
```

Comment:
The traininfo files contain 2 fields, the ID (aka system name) and the Name (aka a sort of display name, but not quite)
There are 2 versions of the file v1 & v2. Neither will read correctly if you make these changes, necessitating the conversion, in may case of about 30 files, and in the clubs case a few hunderd). If you want to do this you need to add a version 3, you need to go from top to bottom of the dispatcher code to ensure that nowhere is it using explicitly a systemname.

Reply:
Thank you for the insight into train info versions.  I have implemented version 3.  The load process takes care of converting the transit and block names for either version 1 or version 2.

I also changed the log statements to use the USERNAME_SYSTEMNAME DisplayOption instead of DISPLAYNAME.  They were originally just the system name.

Reply:
I will take the changes for a trip down the main line 2morrow EDT. Thx for the changes.

Reply:
Looks good. Thx. Using getDisplayName does the job, I just have trouble thinking of that as getObjectsUniqueIdentifier.... getting old.